samples/net/dhcpv4_client/prj.conf

```diff
@@ -22,14 +27,14 @@ CONFIG_SYS_LOG=y
 CONFIG_SYS_LOG_NET_LEVEL=4
 CONFIG_SYS_LOG_SHOW_COLOR=y
 
-CONFIG_NET_DEBUG_CORE=n
+CONFIG_NET_DEBUG_CORE=y
 CONFIG_NET_DEBUG_IF=n
-CONFIG_NET_DEBUG_IPV4=n
+CONFIG_NET_DEBUG_IPV4=y
 CONFIG_NET_DEBUG_UDP=n
-CONFIG_NET_DEBUG_DHCPV4=n
-CONFIG_NET_DEBUG_CONN=n
+CONFIG_NET_DEBUG_DHCPV4=y
+CONFIG_NET_DEBUG_CONN=y
 CONFIG_NET_DEBUG_ARP=n
 CONFIG_NET_DEBUG_L2_ETHERNET=n
 
-CONFIG_SLIP_DEBUG=n
+CONFIG_SLIP_DEBUG=y
```

Comment:
Please do not enable the debugs in default prj.conf file. If you need debugging prints, create a conf file just for this purpose and use that in your testing.